code/turf/turf_flock.dm

```diff
@@ -324,7 +324,7 @@
 	..()
 	if(!istype(F) || !oldloc)
 		return
-	if(F.client && F.client.check_key(KEY_RUN) && !F.floorrunning && F.resources >= 1)
+	if(!F.floorrunning && F.resources >= 1)
```

Comment:
Is turf Entered() called whenever set_loc() is called passing turf or something else?

Reply:
`set_loc()`

Reply:
Couldn't there be possible issues, such as wormholes, making you auto-floorrun if you teleport into a wall without a key check?